src/components/SelectUser/SelectUser.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { User } from '../../types/Users';
+import { DefaultErrorsState } from '../../types';
+
+type Props = {
+  name: string;
+  label: string;
+  value: string;
+  users: User[];
+  onChange: (newValue: string) => void;
+  selectId: string;
+  errors: DefaultErrorsState;
+};
+export const SelectUser: React.FC<Props> = ({
+  label,
+  selectId,
+  name,
+  value,
+  users,
+  onChange,
+  errors,
+}) => {
+  const handleChangeSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    onChange(event.target.value);
+  };
+
+  return (
+    <div className="field">
+      <label htmlFor={`${name}-${selectId}`}>{label}</label>
+      <select
+        id={`${name}-${selectId}`}
+        data-cy={name}
+        onChange={handleChangeSelect}
+      >
+        <option value={value} disabled selected>
```

Comment:
The `selected` attribute in the default option is unnecessary and can cause issues with React's controlled component behavior. The `value` prop on the `<select>` element already controls the selected option. Consider removing the `selected` attribute to avoid conflicts.